### PR TITLE
fix PopoverProps type error and update unit test

### LINKED ISSUE #46364: popover缺失forceRender
### Reproduction link

[https://github.com/ant-design/ant-design](https://github.com/ant-design/ant-design)

### Steps to reproduce

无

### What is expected?

popover支持forceRender

### What is actually happening?

不支持

| Environment | Info |
| --- | --- |
| antd | 5.12.1 |
| React | 18.2.0 |
| System | macos 14 |
| Browser | chrome |

### PR DESCRIPTION
**Pull request by liangkuku**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [x] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link
close [46364](https://github.com/ant-design/ant-design/issues/46364)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Fix the issue of unsupported "forceRender" in the ts type of PopoverProps and add corresponding unit tests.  |
| 🇨🇳 Chinese |   修复PopoverProps的ts类型中不支持forceRender的问题，并添加对应单元测试    |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/popover/__tests__/index.test.tsx [hunk 1]
```diff
@@ -89,4 +89,24 @@ describe('Popover', () => {
     );
     expect(Array.from(wrapper.container.children)).toMatchSnapshot();
   });
+
+  it('forceRender works', () => {
+    const content = (
+      <div className='forceRender'>
```
Comment (afc163) [inline components/popover/__tests__/index.test.tsx:95]:
```suggestion
      <div className="forceRender">
```

File: components/popover/index.tsx [hunk 2]
```diff
@@ -14,6 +14,8 @@ import useStyle from './style';
 export interface PopoverProps extends AbstractTooltipProps {
   title?: React.ReactNode | RenderFunction;
   content?: React.ReactNode | RenderFunction;
+  /** Force render Popover content*/
+  forceRender?: boolean;
```
Comment (afc163) [inline components/popover/index.tsx:18]: 看下这里的 lint 提示信息。

Comment (afc163) [reply components/popover/index.tsx:18]: 直接在这里加上 forceRender 更好：https://github.com/react-component/tooltip/blob/8f4ac4f801d9170dfb8423c3a24682d2c2f53630/src/Tooltip.tsx#L12C21-L18

Comment (afc163) [reply components/popover/index.tsx:18]: 不用动，其实这里都不用加 test case。

Comment (liangkuku) [reply components/popover/index.tsx:18]: 已更新[472](https://github.com/react-component/tooltip/pull/472)

**Comment by afc163** [comment]
Fixed in https://github.com/react-component/tooltip/pull/472

**Pull request closed without merging**